app/src/main/res/values-sw600dp-land/dimens.xml

```diff
@@ -197,4 +197,20 @@
   <!-- Previous Responses Item: Question View -->
   <dimen name="previous_responses_item_question_view_margin_start">204dp</dimen>
   <dimen name="previous_responses_item_question_view_margin_end">204dp</dimen>
+
+  <!-- General Button Item: Exploration Split View -->
+  <dimen name="general_button_item_exploration_split_view_margin_start">48dp</dimen>
+  <dimen name="general_button_item_exploration_split_view_margin_end">64dp</dimen>
+
+  <!-- General Button: Exploration View -->
```

Comment:
https://xd.adobe.com/view/dfe8ccd4-39eb-414d-a0b2-b5674357f966-8abb/screen/9b11d6ac-ba50-4d28-b355-28916363cfc9/specs/

Reply:
Correct

Reply:
Looks a bit odd, the start margin is the same as the content, which makes them display in a similar margin of 176, but the end, the button has 176 and the content has 208. 

But, in another exploration view, we tend to keep the button more margin so it comes inside than content.

Reply:
For mobile we are generally keeping the buttons a bit inward w.r.t. overall content but in tablet we are keeping them outward. The outward is more correct but for mobile we need to make sure that there is sufficient margins on all side and thats why we kept it inwards.